bark/world/world.cpp

```diff
@@ -72,7 +72,7 @@ void World::Execute(const double& delta_time) {
   const double inc_world_time = world_time_ + delta_time;
   using models::dynamic::StateDefinition::TIME_POSITION;
   for (auto agent : agents_) {
-    if (agent.second->IsValidAtTime(inc_world_time) &&
+    if (agent.second->IsValidAtTime(world_time_) &&
```

Comment:
what does this change do? was it not working before?

Reply:
that's only relevant when simulating the interaction dataset in closed loop. It's an error which I introduced when refactoring the Execute function. With this change now, it's the same as it was before.